src/_posts/compliance/security-overview/2000-01-01-incident-support.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Incidents and Support
+nav: Incidents & Support
+modified_at: 2024-06-20 00:00:00
+tags: compliance security measures
+index: 12
+---
+
+### Security Incident Response
+
+In the event of a security incident, our operations team follows a predefined incident response plan to contain the
+incident and mitigate its impact. The incident response plan includes procedures for identifying the cause of the
+incident, containing the incident, and restoring the platform to normal operation. After the incident has been resolved,
+we perform a post-mortem to identify the root cause of the incident and to prevent similar incidents from occurring in
+the future.
+
+### Security Incident Reporting
+
+If you suspect a security incident, please report it to our security team at
+[security@scalingo.com](mailto:security@scalingo.com). We take all reports of security incidents seriously and we will
+investigate the incident promptly.
+
+### Security Incident Communication
+
+In the event of a security incident, we will communicate with our customers to keep them informed of the incident and to
+provide updates on the status of the incident. We will provide regular updates on the incident and we will inform our
+customers of any actions they need to take to protect their data. Security communication is done through our
+[status page](https://scalingostatus.com), and by chat support or email if necessary.
+
+### On-Call Rotation
+
+Our operations team is available 24/7 to respond to security incidents and to ensure the availability of the platform.
+We have an on-call rotation to ensure that there is always someone available to respond to security incidents and to
+ensure the availability of the platform. Our on-call first response time is 15 minutes.
```

Comment:
Is it true even by night ? I guess so but we must be sure if we indicate it here

Reply:
How would you change the sentence ?

Reply:
It's not support availability : it's only the SRE operators